contents/handbook/small-teams/experimentation/index.mdx

```diff
@@ -82,3 +82,27 @@ hideAnchor: false
 
 ## Feature ownership
 You can find out more about the features we own [here](/handbook/engineering/feature-ownership)
+
+## Long term vision
+
+Imagine Alice is a product manager, and Bob is an engineer, both of whom love using PostHog. They're building a new feature together, but they're unsure of the impact, so Alice & Bob decide to gate the feature via a feature flag.
+
+Bob runs the PostHog CLI, which automatically converts his branch to a feature flagged version. Alice then sets up auto-rollback on this feature flag using the PostHog UI: If their conversion metric falls more than 20%, the feature must be automatically rolled back.
+
+Bob feels this isn't enough, and also adds a condition for Sentry errors increasing by 30%. This should result in an automatic rollback as well.
```

Comment:
```suggestion
Alice checks the Posthog UI and because it's such an important feature - adds a safety condition for Sentry errors increasing by 30% and a few counter metrics. This should result in an automatic rollback as well. Alice starts the experiment.
```